plugins/ban/bot/main.py

```diff
@@ -311,6 +153,67 @@ async def rban(
                 await ctx.send(f"{user} a bien été banni !")
             await ctx.send("https://thumbs.gfycat.com/LikelyColdBasil-small.gif")
 
+    def load_friendly_ban(self):
+        """Loads configuration and events for the friendly ban command"""
+        
+        # look for the configuration, gets an empty dict if it doesn't exist
+        self.friendly_ban_config = self.bot.config.get("friendly_ban", {})
+
+        # loads the guild ids from the configuration
+        self.friendly_ban_guilds: list[int] = self.friendly_ban_config.get(
+            "guilds",
+            []
+        )
+
+        # loads the events
+        self.friendly_ban_events = self.friendly_ban_config.get("events", [])
+        self.systematic_events: list[function] = []
+        self.random_events: list[function] = []
+
+        for event in self.friendly_ban_events:
+            chances = event.get("chances", None)
+            if chances == None:
```

Comment:
Remplace == None par is None (et idem pour les != None s’il y en a). C’est une PEP, et légèrement plus rapide (genre une ou deux instructions en moins mdr)

Reply:
yep je sais encore une fois c'était lié à un ancien design de code mais quand j'ai changé la valeur j'ai pas fait attention x)